document_page/README.rst

```diff
@@ -0,0 +1,100 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Document Page
+==============
+
+This module allows you to write web pages for internal documentation.
+
+Installation
+============
+
+This module depends on module knowledge. So make sure to have available it in your addons list
+
+Configuration
+=============
+
+No configuration required
+
+Usage
+=====
+
+To use this module, you need to:
+* Install the module's dependencies
+* Install this module module
+* Got to knowledge menu
+* use submenu Categories to Create the document's category you need
+* use submenu Pages to  Create pages for that category
```

Comment:
To use this module, you need to:
* Go to Knowledge menu
* Click on Categories to create the document's category you need with the template
* Click on Pages to create pages and select the previous category to use the template